pages/linux/upower.md

```diff
@@ -0,0 +1,24 @@
+# upower
+
+> System utility to provide battery information and statistics.
+> More information: <https://upower.freedesktop.org/>.
+
+- Dump battery information:
+
+`upower --dump`
+
+- List connected battery-using devices:
```

Comment:
I'm not really happy with this description. When I run that command, I get:
```
.../line_power_AC0
.../battery_BAT0
.../DisplayDevice
```
I wouldn't call `line power` and `battery` battery-using. However, there isn't really good documentation for this and the only description I found was `enumerate all power objects on the system`. 
How about:
```suggestion
- List all power objects on the system:
```

Reply:
That's fine too.